tests/qa_automation/install_test_suite.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary:  the step to install testsuite from QA:Head
+# Maintainer: Yong Sun <yosun@suse.com>
+
+use strict;
+use base "opensusebasetest";
+use utils;
+use testapi;
+
+sub run {
+    select_console('root-console');
+    my $test = get_var('QA_TESTSUITE');
+    my $testsuite = "qa_test_$test";
+    zypper_call("in '$testsuite'");
```

Comment:
I guess QA_TESTSUITE is always needed in the suite?
Introduce get_required_var for you :)

Reply:
yes, it 's always needed